docs/openapi/components/schemas/credentials/MillTestReportCertificate.yml

```diff
@@ -478,8 +478,8 @@ example: |-
                 "type": [
                   "MechanicalProperty"
                 ],
-                "identifier": "",
-                "name": "Gauge Length"
+                "identifier": "ISO 1352:2021",
```

Comment:
https://github.com/w3c-ccg/traceability-vocab/blob/e92bb49a71b74c3d05fe75dc76ef1e99fdde3d80/docs/openapi/components/schemas/common/SteelProduct.yml#L119
These seem to have a minor difference in them, probably a good idea for them to be identical